oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
<p>Remember about the pie charts when you are going to display proportions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>